main.development.js

```diff
@@ -33,71 +77,24 @@ app.on('ready', () => {
         height: 728
     });
 
-    // TODO: solve issue #58 before uncommenting this code
-    // const logToFile = bunyan.createLogger({
-    //     name: 'plotly-database-connector-logger',
-    //     streams: [
-    //         {
-    //             level: 'info',
-    //             path: OPTIONS.logpath
-    //         }
-    //     ]
-    // });
-
-    function log(logEntry, code = 2) {
-
-        // default log detail set to 1 (warn level) in ./args.js
-        if (code <= OPTIONS.logdetail) {
-            // switch (code) {
-            //     case 0:
-            //         logToFile.error(logEntry);
-            //         break;
-            //     case 1:
-            //         logToFile.warn(logEntry);
-            //         break;
-            //     case 2:
-            //         logToFile.info(logEntry);
-            //         break;
-            //     default:
-            //         logToFile.info(logEntry);
-            // }
+    setupHTTP({sequelizeManager, serverMessageReceive, mainWindow, OPTIONS});
 
-            if (!OPTIONS.headless) {
-                mainWindow.webContents.send(channel, {
-                    log: {
-                        logEntry,
-                        timestamp: timestamp()
-                    }
-                });
-            }
-
-        }
+    // TODO: shell scripts for HTTPS setup may not work on windows atm
+    if (
+        process.platform === 'darwin' &&
+        !contains('--test-type=webdriver', process.argv.slice(2))
+    ) {
+        setupHTTPS(
+            {sequelizeManager, serverMessageReceive, mainWindow, OPTIONS}
+        );
     }
 
-    const sequelizeManager = new SequelizeManager(log);
-
     // clear the log if the file existed already and had entries
-    // clearLog();
-
-    const server = restify.createServer();
-
-    sequelizeManager.log(`Setting up server on port ${OPTIONS.port} ...`, 2);
-
-    server.use(restify.queryParser());
-    server.use(restify.CORS({
-        origins: ['*'],
-        credentials: false,
-        headers: ['Access-Control-Allow-Origin']
-    })).listen(OPTIONS.port);
-
-    sequelizeManager.log('Done.', 2);
-
-    sequelizeManager.log('Rendering the app...', 2);
+    clearLog();
```

Comment:
I don't think that we want to clear the log files when the app restarts. When the app is running headlessly on the server it may crash and automatically restart when something goes wrong - and it is these logs that will tell us what happened.

Instead, we should make sure to include a log entry that is like "App starting" so that we can distinguish between sessions.